dev/com.ibm.ws.org.eclipse.equinox.console/bnd.overrides

```diff
@@ -18,7 +18,7 @@ Private-Package: \
    org.eclipse.equinox.console.telnet
 
 -includeresource: \
-   @${repo;org.eclipse.equinox.console;1.1.200.v20150929-1405}!/about.html, \
```

Comment:
The bundle is identical with and without this change.  Since `;EXACT` was not added to the `repo;`, it will use that level of a later one if available.  Since there is only the newer one in the `.maven` files and in the `-buildpath` that is the one that is used.